### DISCUSSION TITLE
Changing Display Currency

### DISCUSSION THREAD

**Discussion by oliverlabs** [mivano/azure-cost-cli#94] [Ideas]
Is it possible to change currency from a default billing currency to a custom one? I know there is a `--useUSD` option, but does Azure Cost API allow for a custom currency display option?

**Comment by mivano** [comment]
At the moment, it fetches the configured currency (the one reported/configured on your subscription by Azure) and the USD variant. That is what is returned by the API. Not all the APIs support specifying another currency. The `useUSD` parameter forces the (console/text/markdown) output to display the USD one. In most other exports, you will see both your currency and the USD being returned. 

So the answer is no; you cannot indicate another currency.

**Reply by oliverlabs** [reply]
Thanks for answering, @mivano.
